examples/tpm2-ekcert/main.go

```diff
@@ -0,0 +1,61 @@
+// Binary tpm2-ekcert reads an x509 certificate from a specific NVRAM index.
+package main
+
+import (
+	"crypto/x509"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/google/go-tpm/tpm2"
+	"github.com/google/go-tpm/tpmutil"
+)
+
+var (
+	tpmPath = flag.String("tpm-path", "/dev/tpm0", "Path to the TPM device (character device or a Unix socket)")
+	// Default value is defined in section 7.8
+	// https://trustedcomputinggroup.org/wp-content/uploads/TCG-TPM-v2.0-Provisioning-Guidance-Published-v1r1.pdf
+	certIndex = flag.Uint("cert-index", 0x01C00002, "NVRAM index of the certificate file")
+	outPath   = flag.String("output", "", "File path for output. Leave blank to write to stdout.")
```

Comment:
tiny nit: inconsistent flag help text: there's a period at the end of the text for outPath but not for the others. I think the standard style is to not have punctuation at the end of flag help text.

Reply:
Done